atef/config.py

```diff
@@ -1410,3 +1414,23 @@ def get_result_from_comparison(
         )
 
     return item, item.result
+
+
+async def run_passive_step(
```

Comment:
Small thought: is this the right module to put this coroutine?

Reply:
Maybe we might be missing `atef.prepared` to separate out 'checks' vs 'configs' vs 'prepared configs' vs 'widgets/pages'?  That sounds like... too much work.